index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    
+    <header id="header">
+      
+        <img width="300px" src="https://s3.amazonaws.com/freecodecamp/original_trombones.png" alt="picture" id="header-img">
+        
+        <nav id="nav-bar">
+          
+            <a class="nav-link" href="#features">Features</a></li>
+            <a class="nav-link" href="#howitworks">How it Works</a></li>
+            <a class="nav-link"  href="#pricing">Pricing</a></li>
+        
+          
+      </nav>
+      
+      
+      
+    </header>
+  
+    
+    
+    <section id="features">
+        <h1 class="form-text">Handcrafted, home-made masterpieces</h1>
+        <form id="form"  action="https://www.freecodecamp.com/email-submit">
+          <input type="email" id="email" name="email" required placeholder="Enter your email adress">
+          <input button type="submit" id="submit" value="GET STARTED">
+      
+        </form>
+      <div class="prem-mat">
+      <h1>Premium Materials</h1>
+      <h4>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase</h4> 
+    </div>
+    <div class="prem-mat">
+      <h1>Fast Shipping</h1>
+      <h4>We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied.</h4> 
+    </div>
+    <div class="prem-mat">
+      <h1>Quality Assurance</h1>
+      <h4>For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument.</h4> 
+    </div>
```

Comment:
Avoid using so many h1 tags in your HTML document. visit this site https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md for HTML and CSS best practices.